modules/mono/editor/GodotTools/GodotTools/GodotSharpEditor.cs

```diff
@@ -375,6 +378,8 @@ public override void _EnablePlugin()
         {
             base._EnablePlugin();
 
+            Connect("project_settings_changed", new Callable(this, "RefreshProjectLocation"));
```

Comment:
Is the setting not available as an event?

```suggestion
            ProjectSettingsChanged += GodotSharpDirs.DetermineProjectLocation;
```

Reply:
Oops. Sorry. It is of course. :no_mouth: